lib/Runtime/Debug/DebugContext.h

```diff
@@ -37,6 +37,9 @@ namespace Js
         // The debugger is running which means that the engine is
         // running in interpreted mode.
         Debugging,
+
+        // The debugger is running but recording is suspended.
+        RecordingSuspended,
```

Comment:
I would suggest not to add a DebuggerMode but use a flag on DebugContext the way kyle did for isReparsingSource
[https://github.com/Microsoft/ChakraCore/commit/1c2d2591d4a73415d71d468264dd8e5676862255](https://github.com/Microsoft/ChakraCore/commit/1c2d2591d4a73415d71d468264dd8e5676862255)